products/ubuntu.md

```diff
@@ -118,16 +117,39 @@ releases:
+Packages in `universe` are expected to be community-supported on a best-effort basis during the LTS phase, but in practice, security-updates are only released to pro subscribtions.
+
+Extended Security Maintenance (ESM) provides security updates on Ubuntu LTS releases for additional 5 years. It is available with the [Ubuntu Pro](https://ubuntu.com/pro) subscription or a [Free subscription for personal use](https://ubuntu.com/blog/ubuntu-pro-beta-release)[^4].
+
+Ubuntu Pro offers security fixes for critical, high, and selected medium CVEs in the `main` and `universe` repositories. Ubuntu Pro (Infra-only)[^1] only guarantees security fixes for packages in the `main` repository.
+
+## Support Comparison
 
-The dates for active and security support are taken from [here](https://github.com/canonical-web-and-design/ubuntu.com/blob/master/static/js/src/chart-data.js) what is used for the graph rendering on the [Release Cycle Page](https://ubuntu.com/about/release-cycle).
+Plan | Ubuntu LTS | Ubuntu Pro (Infra-Only) [^1] | Ubuntu Pro
+-----|------------|------------------------------|-----------
+Main repository | 5 years | 10 years             | 10 years
+Restricted repository | 5 years | 10 years[^2]   |10 years [^2]
+Universe repository | Best Effort | Best Effort | 10 years 
+Phone/Ticket Support | No | Yes | Yes
+Kernel Live Patching | No  | Yes | Yes
+[Security Certifications and Hardening](https://ubuntu.com/security/certifications)[^3] | No | Yes | Yes
+[Ubuntu Landscape](https://ubuntu.com/landscape) | No | Yes | Yes
 
-Ubuntu Pro offers security fixes for critical, high, and selected medium CVEs in the `main` and `universe` repositories. Ubuntu Pro (Infra-only) - previously known as Ubuntu Advantage - only guarantees security fixes for packages in the `main` repository - other packages are on a best-effort basis.
+For package specific support details, the following commands are available:
+
+- `ubuntu-security-status` command (`ubuntu-support-status` on versions before `20.04`) for non-ESM.
+- [`pro security-status`](https://manpages.ubuntu.com/manpages/kinetic/en/man1/ubuntu-advantage.1.html) for ESM.
+
+## Release Cadence
+
+Releases of Ubuntu get a development codename ("Breezy Badger") and are versioned by the year and month of delivery - for example Ubuntu 17.10 was released in October 2017. LTS or "Long Term Support" releases are published every two years in April. Every six months between LTS versions, Canonical publishes an interim release of Ubuntu. See [this link](https://ubuntu.com/about/release-cycle) for more details on the Ubuntu Release Cycle.
 
-For package specific support details, run the `ubuntu-security-status` command (`ubuntu-support-status` on versions before `20.04`).
+[^1]: previously known as Ubuntu Advantage.
```

Comment:
I like the footnotes but unfortunately they are not displayed at a suitable location. With the current rendering it looks like it belongs to the _Release Cadence_ section. This issue is similar to the "More information is available on..." issue : it should be clearly separated from the description.

Reply:
I've added a `<hr>` at the bottom of the main text. Not great, but slightly better, perhaps?

https://deploy-preview-2424--endoflife-date.netlify.app/ubuntu

<img width="585" alt="image" src="https://user-images.githubusercontent.com/584253/217734549-2952f19b-e949-49bd-b4f3-af763b1d8ab7.png">

Reply:
Yep, slightly better. Anyway I do not see another way around it for the time being, so...